tests/_support/Page/bill/Create.php

```diff
@@ -38,6 +38,13 @@ public function fillMainBillFields(array $billData): void
         (new Input($I, '#billform-0-quantity'))->setValue($billData['quantity']);
     }
 
+    public function fillBillDescriptionField(string $descr, int $billNo = 0): void
```

Comment:
```suggestion
    public function fillBillDescriptionField(int $billNumber, string $descr): void
```
не може буть не обов*язковим номер біла якому ти вписуєш дескріпшн, це з назви метода не логічно

Reply:
Ну тип если создается/апдейтится/копируется билл, то там одно поле - #billform-0-label, зачем тогда задавать руками?. 
ну а если правятся два - то там #billform-0-label #billform-1-label и тп. И там уже над их различать.

Reply:
это не номер самого била, а его место на страничке правки
![image](https://user-images.githubusercontent.com/83531949/135244952-f6b8fa22-3c2c-4aa9-9a8f-11164da59d20.png)

Reply:
у тебе цей метод лежить в класі креейт, він не знає скільки ти хочеш білів створювать. типу якщо метод заповнює всі біли, то він прийме всі біли, якщо метод заповнює поля по одному білу то він прийме поля по одному білу, а якщо ти хочеш метод який заповнює поле для конкретного біла, то треба і то і то поле задавать